test/e2e/tls_util_test.go

```diff
@@ -234,3 +271,25 @@ func TestOnlyCAExist(t *testing.T) {
 		t.Fatalf("expect %+v, but got %+v", appSecret, appSecretFromCluster)
 	}
 }
+
+// use Pod as a dummy runtime object for the CR input of GenerateCert().
+func toDummyCR(namespace string) runtime.Object {
```

Comment:
Rename this to `newDummyCR()` since we're not really converting anything to use `toDummyCR()`.
Also change the comment format to be Go style: `newDummyCR returns a dummy runtime object ....`

Reply:
yeah, that's right.

Reply:
fixed